WebContent/WEB-INF/web.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
+	id="WebApp_ID" version="4.0">
+	<display-name>Book - Management</display-name>
+
+	<welcome-file-list>
+		<welcome-file>view/Welcome.jsp</welcome-file>
+	</welcome-file-list>
+
+	<servlet>
+		<servlet-name>Book</servlet-name>
+		<servlet-class>com.codeenginestudio.bookManagement.servlets.BookServlet</servlet-class>
```

Comment:
The package name shouldn't include the uppercase word! suggest `com.codeenginestudio.management.book.servlets`